src/components/DataTable/DataTable.tsx

```diff
@@ -213,6 +213,7 @@ export function DataTable<T>({
               <DataTableRow key={headerGroup.id}>
                 {headerGroup.headers.map((header) => (
                   <th
+                    className={styles['data-table__header-cell-container']}
```

Comment:
Not a complaint, but curious about this naming format that mizes underscore and dash

Reply:
Yep yep, [doc.s here](https://chanzuckerberg.github.io/edu-design-system/?path=/docs/documentation-contributor-guidelines-code-guidelines--docs)

In our scheme, `-` has no special meaning and is in place of using camelCase in CSS naming, preferring snake-case. `--` and `__` have more strict meaning, with most of the rest is convention

HOLD ME ACCOUNTABLE to that page :)